README.Rmd

```diff
@@ -38,7 +40,7 @@ This set of functions/package accesses data from:
 	+ Relavant functions:
```

Comment:
This PR looks very nice. Could you also fix this typo of "Relavant" to "Relevant" here?

Reply:
Thanks for the pointer! Done. 
Also added this snippet to the `README.Rmd` file: 
```
---
output: github_document
---
<!-- README.md is generated from README.Rmd. Please edit that file -->

```
This way the Rmd file is rendered automatically to a md. file